### ISSUE TITLE
 1.15.3-1~dev~ubuntu18.04~ppa4 systemctl crio.service failed missed /usr/libexec/crio/conmon directory

### ISSUE THREAD

**Issue by Slach** [cri-o/cri-o#2903]
**Description**
crio.service failed start cause /usr/libexec/crio/conmon missed on debian package

**Steps to reproduce the issue:**
```
apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 018BA5AD9DF57A4448F0E6CF8BECF1637AD8C79D
echo "deb http://ppa.launchpad.net/projectatomic/ppa/ubuntu bionic main" > /etc/apt/sources.list.d/crio.list
apt-get install cri-o-1.15
systemctl start crio.service
```

**Describe the results you received:**
```
-- Unit crio.service has begun starting up.
Oct 20 05:25:01 ubuntu-bionic crio[2899]: time="2019-10-20 05:25:01.955432575Z" level=fatal msg="library config validation: runtime config: invalid conmon path: 
stat /usr/libexec/crio/conmon: no such file or directory"
Oct 20 05:25:01 ubuntu-bionic systemd[1]: crio.service: Main process exited, code=exited, status=1/FAILURE
Oct 20 05:25:01 ubuntu-bionic systemd[1]: crio.service: Failed with result 'exit-code'.
Oct 20 05:25:01 ubuntu-bionic systemd[1]: Failed to start Container Runtime Interface for OCI (CRI-O).
-- Subject: Unit crio.service has failed
```

**Describe the results you expected:**
successfull started cri-o

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `crio --version`:**
```
root@ubuntu-bionic:/home/vagrant# crio --version
crio version 1.15.3-dev
```

dear @lsm5 i hope it will be easy fixed ;-)

**Comment by pierreozoux** [comment]
Can I add a regression test [here}(https://github.com/cri-o/cri-o/blob/master/.circleci/config.yml#L262) to avoid this in the future?

**Comment by haircommander** [comment]
please feel free to submit a PR :)

**Comment by rhatdan** [comment]
@pierreozoux We love adding tests.

**Comment by arashkaffamanesh** [comment]
I'm hitting the same issue:
```
~# systemctl start crio.service
A dependency job for crio.service failed. See 'journalctl -xe' for details.
```

Here is the  `journalctl -xe` output:
https://pastebin.com/FZc0ziSS

**Comment by afbjorklund** [comment]
Looks like it lives in `/usr/bin/conmon` now:

http://ppa.launchpad.net/projectatomic/ppa/ubuntu/pool/main/c/conmon/conmon_2.0.2-1~ubuntu18.04~ppa1_amd64.deb

**Comment by afbjorklund** [comment]
@arashkaffamanesh : your issue is different, it's about the crio-wipe service:

```
-- Unit crio-wipe.service has begun starting up.
Nov 03 10:24:16 master crio[16393]: time="2019-11-03 10:24:16.246397164+01:00" level=fatal msg="command \"wipe\" not supported"
Nov 03 10:24:16 master systemd[1]: crio-wipe.service: Main process exited, code=exited, status=1/FAILURE
Nov 03 10:24:16 master systemd[1]: crio-wipe.service: Failed with result 'exit-code'.
```

**Comment by arashkaffamanesh** [comment]
@afbjorklund THX!

I built from source as follow and it works :-)

```
multipass launch ubuntu --name master --cpus 2 --mem 2G --disk 8G
multipass shell master
sudo -i
apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 018BA5AD9DF57A4448F0E6CF8BECF1637AD8C79D
echo "deb http://ppa.launchpad.net/projectatomic/ppa/ubuntu bionic main" > /etc/apt/sources.list.d/crio.list
apt-add-repository ppa:projectatomic/ppa -y
apt-get update -qq && apt-get install -y runc conmon btrfs-tools containers-common git golang-go libassuan-dev libdevmapper-dev libglib2.0-dev libc6-dev libgpgme11-dev libgpg-error-dev libseccomp-dev libsystemd-dev libselinux1-dev pkg-config go-md2man cri-o-runc libudev-dev software-properties-common gcc make
git clone https://github.com/cri-o/cri-o
go get github.com/cpuguy83/go-md2man
cd cri-o/
make BUILDTAGS=""
make install
make install.systemd
mkdir /etc/crio
cp /root/cri-o/crio.conf /etc/crio/
systemctl daemon-reload
systemctl start crio
systemctl status crio.service
```

**Comment by red-trela** [comment]
So this issue is indeed fixed by a simple:
`sudo sed 's|/usr/libexec/crio/conmon|/usr/bin/conmon|' -i /etc/crio/crio.conf`

Could the package please be fixed accordingly?

**Comment by ToroNZ** [comment]
In Clear Linux, I had to install conmon using the container basic dev bundle:

```
sudo swupd bundle-add containers-basic-dev
sudo sed 's|/usr/libexec/crio/conmon|/usr/libexec/podman/conmon|' -i /etc/crio/crio.conf
sudo systemctl restart crio
```

**Comment by Slach** [comment]
solved in new crio-o-1.17
installed from https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/
